packages/millicast-sdk/src/View.js

```diff
@@ -254,8 +254,10 @@ export default class View extends BaseWebRTC {
     await webRTCPeerInstance.createRTCPeer(this.options.peerConfig)
     // Stop emiting events from the previous instances
     this.stopReemitingWebRTCPeerInstanceEvents?.()
+    this.stopReemitingSignalingInstanceEvents?.()
     // And start emitting from the new ones
     this.stopReemitingWebRTCPeerInstanceEvents = reemit(webRTCPeerInstance, this, Object.values(webRTCEvents).filter(e => e !== webRTCEvents.track))
+    this.stopReemitingSignalingInstanceEvents = reemit(signalingInstance, this, [signalingEvents.broadcastEvent])
```

Comment:
We should not call reemit here since we already hooked these event in 'initConnection' and will reemit these events manually after processing them

Reply:
So @vincentsong remove this line, but keep line 257? Is that the suggested change?

Reply:
@vincentsong The latest change should do the trick. The problem was that where the new listener was emitting the events, it was only emitting them when `event.data.sourceId` is null. I moved the emitter outside the if condition. Let's discuss if the change between the return and break it's OK or the emit in that case should be before the while loop

Reply:
I think I answered my question, will put the emitter back before the while loop